apps/heft/src/schemas/templates/clean.json

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Should these go under `schemas`? Or should they go under `includes` or something?

Reply:
Let's put these under `src/templates`